app/components/Form/Validators/index.js

```diff
@@ -13,3 +13,15 @@ export const NAME_PATTERN = ENGLISH_LETTERS_PATTERN;
 export const EMAIL_ALLOWED_PATTERNS = EMAIL_ADDRESS_ALLOWED_CHARS;
 
 export const PHONE_NUMBER_PATTERN = /^[0-9]*$/;
+
+export const minLengthNameFieldvalidator = (message) => (value) => value && value.length < 3 ? message : '';
```

Comment:
- - Instead of checking length of each field with different validator. Can we have a common validator? 
Say **lengthCheckValidator** which takes _message_, _minLength_ and _maxLength_ in parameters and return error message if _value.length < minLength_ or _value.length > maxLength_.


```suggestion
export const lengthCheckValidator = (message, minLength = 0, maxLength = 25) => (value) => value && (value.length < minLength || value.length > maxLength) ? message : '';
```